### ISSUE TITLE
`sendTo` not directly importable from xstate

### ISSUE THREAD

**Issue by devanfarrell** [statelyai/xstate#2923]
**Description**
The `sendTo` action was introduced in `4.27.0` per the [release notes](https://github.com/statelyai/xstate/releases/tag/xstate%404.27.0) but is not usable without importing from `xstate/lib/actions`

**Expected Result**
`sendTo` should be defined in at least one of these scenarios.

``` js
import { actions } from 'xstate';
const { sendTo } = actions;
```
``` js
import { sendTo } from 'xstate';
```

**Actual Result**
`sendTo` is undefined when imported as done above.

**Additional context**
Version `4.27.0`

The below does work but I believe it was omitted as an oversight.
```js
import { sendTo } from 'xstate/lib/actions'
```

Likely the change needs to be made here. https://github.com/statelyai/xstate/blob/a26897174a85672e70cba788a717c95b53b26235/packages/core/src/index.ts#L7-L25

**Comment by devanfarrell** [comment]
I'm willing to take this on. Only question is if it should be accessible in the former or both access patterns?
``` js
import { actions } from 'xstate'
actions.sendTo()
```
``` js
import { sendTo } from 'xstate'
sendTo()
```

**Comment by davidkpiano** [comment]
> I'm willing to take this on. Only question is if it should be accessible in the former or both access patterns?
> 
> ```js
> import { actions } from 'xstate'
> actions.sendTo()
> ```
> 
> ```js
> import { sendTo } from 'xstate'
> sendTo()
> ```

Would love a PR for this. I'd choose the first pattern `import { actions } from 'xstate'`

**Comment by devanfarrell** [comment]
Verified fixed in [4.28.0](https://github.com/statelyai/xstate/releases/tag/xstate%404.28.0)
